src/zenml/cli/secret.py

```diff
@@ -92,56 +96,10 @@ def register_secret(
 
     Use this command to store sensitive information into a ZenML secret. The
     secret data consists of key-value pairs that can be configured interactively
-    (if the `--interactive` option is set) or via command line arguments.
+    (if the `--interactive` option is set) or via command-line arguments.
     If a schema is indicated, the secret key-value pairs will be validated
     against the schema.
 
-    When passed as command line arguments, the secret field values may also be
```

Comment:
Does this text just get deleted? Are we moving it somewhere else?

Reply:
Ah, that was deleted with the old way cause it didn't make sense to show it in CLI. I return it back until figure out where that should be written.